File: python/412. Fizz Buzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        result = []
        for i in range(1, n+1):
            currentStr = ''
            if(i % 3 == 0):
                currentStr += 'Fizz'
            if(i % 5 == 0):
                currentStr += 'Buzz'
            if(not currentStr):
                currentStr += str(i)
            result.append(currentStr)
        return result

File: javascript/412. Fizz Buzz.js
/**
 * @param {number} n
 * @return {string[]}
 */
var fizzBuzz = function(n) {
    let result = [];
    for(let i = 1; i < n+1; i++){
        let currentString = '';
        if(i % 3 === 0){
            currentString += 'Fizz';
        }
        if(i % 5 === 0){
            currentString += 'Buzz';
        }
        if(!currentString){
            currentString = i.toString();
        }
        result.push(currentString);
    }
    return result;
};